src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/DebugTabView.tsx

```diff
@@ -131,7 +137,7 @@ function DebugTabView({selectedTab = '', chatTabBrickRoad, activeWorkspaceID}: D
 
     const navigateTo = useCallback(() => {
         if (selectedTab === SCREENS.HOME && !!chatTabBrickRoad) {
-            const report = getChatTabBrickRoadReport(activeWorkspaceID);
+            const report = getChatTabBrickRoadReport(activeWorkspaceID, currentReportID, reports, betas, policies, priorityMode, transactionViolations);
```

Comment:
Could we use chatTabBrickRoad directly instead of getChatTabBrickRoadReport function?

Reply:
`chatTabBrickRoad` doesn't include the report that I need to redirect the user

Reply:
getChatTabBrickRoad return a report, right?

<img width="773" alt="Screenshot 2024-10-31 at 11 19 16" src="https://github.com/user-attachments/assets/9a5330b3-51cd-4339-90ff-f0addab4acbf">

Reply:
@DylanDylann nope, it returns a `BrickRoad`